lib/snapshot/snapshot_fuchsia.dart

```diff
@@ -11,5 +11,6 @@ import 'dart:fidl.internal';
 import 'dart:io';
 import 'dart:isolate';
 import 'dart:math';
+import 'dart:mozart.internal';
```

Comment:
we should document dart:mozart.internal and dart:fidl.internal on our wiki somewhere